graphrag/config/models/chunking_config.py

```diff
@@ -26,3 +26,11 @@ class ChunkingConfig(BaseModel):
     encoding_model: str = Field(
         description="The encoding model to use.", default=defs.ENCODING_MODEL
     )
+    prepend_metadata: bool = Field(
+        description="Prepend metadata into each chunk.",
+        default=defs.GRAPHRAG_CHUNK_PREPEND_METADATA,
+    )
+    count_tokens_with_metadata: bool = Field(
```

Comment:
This naming could be a little clearer. I don't particularly like "tokens" also since the chunking may not be tokens. What about "chunk_size_includes_metadata"?